File: Java/프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int plus=0;
        int mul=1;
        for(int i=0; i<num_list.length;i++){
            if(num_list.length >=11) {
                plus+=num_list[i];
                answer=plus;
            }
            if(num_list.length <=10) {
                mul*=num_list[i];
                answer=mul;
            }
        }
        return answer;
    }
}

File: JavaScript/프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.js
function solution(num_list) {
    var answer = 0;
    if(num_list.length>=11){
        for(var i=0; i<num_list.length; i++){
            answer+=num_list[i];
        }
    }
    else {
        answer=1;
        for(var i=0; i<num_list.length; i++){
            answer*=num_list[i];
        }
    }
    return answer;
}